views/batch-deletion/batch-deletion.hbs

```diff
@@ -0,0 +1,32 @@
+{{#extend "lib/loggedin"}}
+
+  {{#content "styles" mode="append"}}
+    <link rel="stylesheet" href="/styles/batch-deletion/batch-deletion.css" />
+  {{/content}}
+  {{#content "scripts" mode="append"}}
+    <script
+      src="/scripts/batch-deletion.js"
+      type="text/javascript"
+      defer
+    ></script>
+  {{/content}}
+
+  {{#content "page"}}
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-12 file-input-container">
+          <form id="batchDeletionFileUploadForm">
```

Comment:
Can we just use `<form action="">` and submit the form without Javascript? Especially when we want to redirect after submitting the form this might be a good choice.